README.md

```diff
@@ -1,20 +1,145 @@
-# suomifi-design-tokens
+# Suomifi Design System Tokens
 
-Design System tokens
+## ✨ Features
 
-The Design system tokens are represented in tokens.json file.
-The file consists of categories and tokens.
+Suomifi Design System Tokens are available in SCSS and JavaScript formats. Typings for TypeScript are also available.
 
-Categories include token prefix and description information.
-All tokens belong to one category.
+The following token types are currently available:
 
-All tokens have a category, version, value, type and comments.
+- colors
+- typography
+- spacing
 
-Token values are defined by type in the following way:
+# 📦 Install
 
-- **hsl**: object with keys for **h**, **s** and **l** and corresponding _integer_ values
-- **font**: object with keys for font_Family, fontSize, lineHeight and fontWeight
-  - **fontFamily**: _array of strings_
-  - **fontSize** and **lineHeight**: _object_ with keys for value and unit where **value** is an _integer_ and **unit** either a _string_ or _null_
-  - **fontWeight**: _integer_
-- **size**: _object_ with keys for value and unit where **value** is an _integer_ and **unit** is either a _string_ or _null_.
+```bash
+yarn add suomifi-design-tokens
+```
+
+# 🔨 Usage
+
+## SCSS
+
+Tokens are available as named SCSS variables and mixins with **\$fi-** -prefix.
+Tokens can be imported to scss files:
+
+```scss
+@import "~suomifi-desing-tokens/tokens";
+```
+
+### Colors
+
+Color tokens are available as variables with additional **colors-** prefix. Colors are provided as hsl values.
+
+```scss
+.custom-class {
+  background-color: $fi-color-brand-base;
+}
+```
+
+### Typography
+
+Typography tokens are available as mixins with additional **typograhpy-** -prefix.
```

Comment:
The hyphen in front of the -prefix is probably redundant (here and in similar places). Would be easier to read without them especially with the prefixes themselves having hyphens in the end.

Reply:
Fixed.